.gitmodules

```diff
@@ -0,0 +1,12 @@
+[submodule "web_video_server"]
+    path = robot/rospackages/src/web_video_server
+    url = https://github.com/RobotWebTools/web_video_server
+[submodule "cv_camera"]
+	path = robot/rospackages/src/cv_camera
+	url = https://github.com/OTL/cv_camera
+[submodule "tf2_web_republisher"]
+	path = robot/rospackages/src/tf2_web_republisher
+	url = https://github.com/RobotWebTools/tf2_web_republisher
+[submodule "rover2018-elec"]
```

Comment:
We should add somewhere in the README a hyper link to `robotics-prototype/rover2018-elec/mod/pds/PDS_code` to make it easier to find

Reply:
done